defaults/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+
+# windows paths (forward slashes for JSON compatiblity)
+sensu_win_base_path: "c:/opt/sensu"
```

Comment:
I wonder if `{{ ansible_user_dir }}` works on windows. If it does, we could do `{{ (ansible_user_dir is defined)|ternary(ansible_user_dir+"/sensu", "c:/opt/sensu") }}` or something. I dunno I'm just spitballing.

Reply:
very fancy! The structure of the tasks is a little cluttered right now... everything is in the windows_client.yml file, but really should be broken out into in the distro specific locations